### PR TITLE
Plugins operators-js

### PR DESCRIPTION
**Pull request by JohannMoller**
Closes #ISSUE_NUMBER

## Checklist

- [ ] Pull request is made to the "develop" branch
- [ ] Tests added
- [ ] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: packages/plugins/operators/operators-js/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,598 @@
+# Change Log
```
Comment (SamTolmay) [inline packages/plugins/operators/operators-js/CHANGELOG.md:1]: Please remove this file

File: packages/plugins/operators/operators-js/src/operators/shared/number.test.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ test('_number valid functions', () => {
   );
   expect(
     _number({ methodName: 'toLocaleString', params: [123456.789, 'de-DE'], location: 'locationId' })
-  ).toBe('123.456,789');
+  ).toBe('123,456.789');
```
Comment (SamTolmay) [inline packages/plugins/operators/operators-js/src/operators/shared/number.test.js:59]: Why has this changed?

File: packages/plugins/plugins/plugin-aws/package.json [hunk 3]
```diff
@@ -51,7 +51,7 @@
     "jest": "27.3.1"
   },
   "peerDependencies": {
```
Comment (SamTolmay) [inline packages/plugins/plugins/plugin-aws/package.json:53]: please remove the peerDependencies section here

File: packages/plugins/operators/operators-js/src/operators/shared/number.test.js (hunk 2 above)
Comment (JohannMoller) [reply packages/plugins/operators/operators-js/src/operators/shared/number.test.js:59]: Not sure. The method returned the new value so I updated it. I'm not sure if the old value passed in the previous tests?

**Review by SamTolmay — APPROVED** [review]

**Pull request merged**